autofill/autofill-impl/src/main/res/layout/fragment_autofill_management_edit_mode.xml

```diff
@@ -34,9 +34,8 @@
             app:type="single_line"
             android:layout_marginBottom="@dimen/keyline_5"
             android:hint="@string/credentialManagementEditLoginTitleHint"
-            android:visibility="gone"
             app:editable="true"
-            app:endIconContentDescription="@string/credentialManagementEditButtonCopyUsername" />
+            app:endIconContentDescription="@string/credentialManagementEditLoginTitleHint" />
```

Comment:
This should just be deleted; we shouldn't have an end icon description here as far as I can see.

Reply:
Deleted